cpp/src/gandiva/encrypt_utils_test.cc

```diff
@@ -20,36 +20,40 @@
 #include <gtest/gtest.h>
 
 TEST(TestShaEncryptUtils, TestAesEncryptDecrypt) {
-  // 8 bytes key
-  auto* key = "1234abcd";
```

Comment:
We drop support for 8 bytes key, right?
(Or 8 bytes key didn't work?)

Reply:
Dropping support since it doesn't match the reference implementations of openssl and hive.